lightning-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lightning-types"
-version = "0.3.0+git"
+version = "0.4.0+git"
```

Comment:
Ideally we won't actually bump the version here in a new release, so maybe we don't touch it until we have a PR breaking semver?

Reply:
True, that's probably preferable. Now force-pushed an update that only bumps `lightning` for now.